### PR TITLE
Extend length of maximum name in settings table

### PR DESCRIPTION
**Pull request by xmacan**
Discusion:

Edit settings_user similarly?

value 75 hardcoded or add somewhere to include/files?

### CODE REVIEW

File: lib/functions.php [hunk 1]
```diff
@@ -417,6 +417,10 @@ function set_config_option($config_name, $value, $remote = false) {
 
 	include_once($config['base_path'] . '/lib/poller.php');
 
+	if (strlen($config_name) > 75) {
+		cacti_log("WARNING: Config option name '$config_name' is too long, will be truncated", false, 'SYSTEM');
```
Comment (netniV) [inline lib/functions.php]: This should be an error not a warning. It creates a problem that can't be fixed by the end user.

**Review by netniV — CHANGES_REQUESTED** [review]

**Review by netniV — APPROVED** [review]

**Comment by netniV** [comment]
Yes settings_user should also be extended to match to avoid issues.

**Comment by xmacan** [comment]
ok, I will do it

**Comment by TheWitness** [comment]
Update the audit schema in the docs folder.

**Comment by netniV** [comment]
Was the schema updated?

**Comment by xmacan** [comment]
My bad, I will fix it

**Comment by TheWitness** [comment]
I fixed it a few days ago.

**Comment by TheWitness** [comment]
Last weekend during my QA testing.

**Pull request merged**